### ISSUE TITLE
Custom "Hits per Level"-Range

### ISSUE THREAD

**Issue by CreepSore** [Frostiae/Flyffulator#66]
As far as I understand the current [`utils.getMonstersAtLevel`](https://github.com/Frostiae/Flyffulator/blob/main/src/calc/utils.js#L151) code, the current displayed monster count is hardcoded. It would be nice to be able to select a range to configure how many entries to display, since as a mage level ~30 for example, you can easily kill monsters 10 to 15 levels above you.

**Comment by Frostiae** [comment]
Yep I agree, that would be nice I can work on that. Could maybe do some kind of slider that shows different ranges of 10 levels or something.

**Comment by Frostiae** [comment]
Obsolete due to 692a9fd3753adb410a68d6379eec59cad9302f6d
